src/ert/_c_wrappers/enkf/config/field_config.py

```diff
@@ -1,9 +1,10 @@
 from cwrap import BaseCClass
 from ecl.grid import EclGrid
-
+import math
+import numpy as np
+import numpy.typing as npt
```

Comment:
consider using `if TYPE_CHECKING` to avoid it being imported at runtime, and `from __future__ import annotations` to avoid having to use ""